app/helpers/bgeigie_imports_helper.rb

```diff
@@ -2,20 +2,20 @@ module BgeigieImportsHelper
 
   def collect_bgeigie_logs(bgeigie_logs)
     bgeigie_logs.collect do |b|
-      point = {
+      {
         :lat => b.latitude, 
         :lng => b.longitude,
         :cpm => b.cpm
       }
     end
   end
 
-  def bgeigie_nav_li(status)
+  def bgeigie_nav_li(status) # rubocop:disable Metrics/AbcSize
```

Comment:
I think 15 was needed. Please check with Robin.

rob



Regards,
Rob Oudendijk Yuka Hayashi
http://yr-design.biz   http://oudendijk.biz  http://about.me/robouden
tel +81 80-22605966  Skype: robouden  Facebook:robouden
<http://on.fb.me/QeKw2P> twitter:robouden <http://bit.ly/RAiSTC>

On Mon, Jun 13, 2016 at 2:03 PM, Eito Katagiri <notifications@github.com>
wrote:

> In app/helpers/bgeigie_imports_helper.rb
> <https://github.com/Safecast/safecastapi/pull/283#discussion_r66740401>:
>
> >          :lat => b.latitude,
> >          :lng => b.longitude,
> >          :cpm => b.cpm
> >        }
> >      end
> >    end
> >
> > -  def bgeigie_nav_li(status)
> > +  def bgeigie_nav_li(status) # rubocop:disable Metrics/AbcSize
>
> I would rather decrease to like 10. I think that the size is currently 15,
> which is quite complex.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Safecast/safecastapi/pull/283/files/d0c3b20d0978d7f9e77b94208ab24757383d857e#r66740401>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABUqtksLufzEZz-ba3ZLfTdPrgBLXJ40ks5qLOSGgaJpZM4IzvV1>
> .
>